contracts/resolvers/DefaultReverseResolver.sol

```diff
@@ -44,5 +48,6 @@ contract DefaultReverseResolver {
      */
     function setName(bytes32 node, string memory _name) public onlyOwner(node) {
```

Comment:
Weren't we going to change to using the public resolver here?

Reply:
I can't remember the exact reason we still have the reverse resolver. I think it was for setting the reverse for the reverse registrar itself?

We pass through the public resolver in setNameForAddr so we are using the public resolver